cocos/network/HttpRequest.h

```diff
@@ -342,6 +336,18 @@ class CC_DLL HttpRequest : public Ref
    		return _headers;
    	}
     
+private:
+    inline void doSetResponseCallback(Ref* pTarget, SEL_HttpResponse pSelector)
```

Comment:
I will cause the function `setResponseCallback` hard to read. I have to dig into the `doSetResponseCallback` function.

I think it's ok to put the implementation of `doSetResponseCallback` directly in `setResponseCallback`